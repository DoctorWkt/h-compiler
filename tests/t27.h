void main()
{
  int i;

  for (i=0; i < 32; i++) {
    if (i > 8 && i < 20)
      printn(i);
  }
  putchar('\n');
  for (i=0; i < 32; i++) {
    if (i == 8 || i == 16 || i==24)
      printn(i);
  }
}
