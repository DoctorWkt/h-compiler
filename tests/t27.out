000011
000012
000013
000014
000015
000016
000017
000020
000021
000022
000023

000010
000020
000030
